Reverse_Linked_List/Reverse_Linked_List.cpp

```diff
@@ -0,0 +1,112 @@
+#include<bits/stdc++.h>
+
+using namespace std;
+
+class Node {
+    /*
+    this class is used to create node
+    ---------
+    | 0  |   |->NULL
+    ---------
+    */
+    public:
+    int data;
+    Node * next;
+    Node(int n) {
+        data = n;
+        next = NULL;
+    }
+};
+
+class LinkedList {
+    private:
+    Node * head;
+    Node * tail;
+    public:
+        LinkedList() {
```

Comment:
Inconsistency in indentation

Reply:
Then, it will become 8 space

Reply:
I resolve it last time, you told me that, It is 8 space

Reply:
Use 4 space instead of 8 space

Reply:
It should be
public:
LinkedList()